application/app/components/fiches/ficheEtablissement/ficheEtablissement.tsx

```diff
@@ -22,11 +22,16 @@ export default function FicheEtablissement({ feature }: FicheEtablissementProps)
 			<br />
 			<ActionButtons />
 			<hr className='my-4' />
-			<ProtectionCard isProtected={!!feature.protection} />
-			<PotentielSolaireCard potentiel_solaire={feature.potentiel_solaire} />
+			{feature.protection && <ProtectionCard />}
+			<PotentielSolaireCard
+				potentiel_solaire={feature.potentiel_solaire}
+				showInterpretation
+				showNbEtablissements={false}
+				nb_eleves={feature.nb_eleves ?? undefined}
+			/>
 			<hr className='my-4' />
 			<div className='ml-2'>
-				<InstallationCard surface_utile={feature.surface_utile} />
+				<InstallationCard surface_exploitable_max={feature.surface_exploitable_max} />
```

Comment:
Les props sont en camelCase !
```suggestion
				<InstallationCard surfaceExploitableMax={feature.surface_exploitable_max} />
```